src/parquet/column/serialized-page.h

```diff
@@ -32,6 +32,10 @@
 
 namespace parquet_cpp {
 
+// 16 MB is the default maximum page header size
+static constexpr int DEFAULT_MAX_PAGE_HEADER_SIZE = 16 * 1024 * 1024;
+// 16 KB is the default expected page header size
+static constexpr int DEFAULT_PAGE_HEADER_SIZE = 16 * 1024;
```

Comment:
Ah let's use `int64_t` here to be safer? On MSVC `int` is 32 bits so this may save us an overflow down the road.